spec/server_spec.rb

```diff
@@ -0,0 +1,61 @@
+require File.expand_path('../spec_helper', __FILE__)
+require 'server'
+require 'octokit'
+require 'api'
+require 'ping_checker'
+require 'signature_verifier'
+require 'rspec'
+require 'rack/test'
+require 'sinatra'
```

Comment:
are you sure this many requires are needed, `spec_helper` should be providing some of these